package.json

```diff
@@ -90,5 +88,11 @@
     "tslint": "~5.11.0",
     "typescript": "~2.9.2",
     "zone.js": "~0.8.26"
+  },
+  "dependencies": {
+    "@fortawesome/angular-fontawesome": "^0.2.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.6",
+    "@fortawesome/free-brands-svg-icons": "^5.4.1",
+    "@fortawesome/free-solid-svg-icons": "^5.4.1"
```

Comment:
previously it was regular. why solid icons are preferred rather than regular? Any Particular reason. It looks nice though.

Reply:
Some of the regular icons are now part of font-awesome Pro, which requires a license e.g. [bold](https://fontawesome.com/icons/bold?style=regular) but they are all available as solid icons for free. Not entirely sure why they have done this for such basic icons...

Reply:
Oh. I was planning to remove few basic operations from the editor, which can be achieved with keyboard shortcuts. i you can tell which icons were not available. I can cross verify with those which I planned to remove. So that If possible we can use the regular font-awesome itself.